front-end/src/ui-components/Address.tsx

```diff
@@ -37,17 +39,29 @@ const Address = ({ address, accountName, className }: Props): JSX.Element => {
 	}, [address, api]);
 
 	return (
-		<div className={className}>
-			<Identicon
-				className='image'
-				value={address}
-				size={32}
-				theme={'polkadot'}
-			/>
-			<div className='content'>
-				<div className='header'>{display || accountName || ''}</div>
-				<div className='description'>{shortenAddress(address)}</div>
-			</div>
+		<div className={displayInline ? `${className} inline`: className}>
+			{displayIdenticon &&
+				<Identicon
+					className='image identicon'
+					value={address}
+					size={displayInline ? 16 : 32}
+					theme={'polkadot'}
+				/>}
+			{displayInline
```

Comment:
Can't we make this more readable by just adding the `inline` conditionnaly:
```js
{display || accountName
	? <div className={displayInline ? 'header inline' : 'header'}>{display || accountName}</div>
	: <div className={displayInline ? 'description inline' :'description'}>{shortenAddress(address)}</div>
```

Reply:
The reason for the nested conditions was that I thought if `Address` is not displayed inline (e.g in Settings) we want to show both name and address.

But if not, I can do the more readable way of course.
<img width="525" alt="Screenshot 2020-03-25 at 17 55 50" src="https://user-images.githubusercontent.com/7072141/77563781-008a5d00-6ec2-11ea-848a-193b72de57d9.png">

Reply:
I think in the code that I mentioned, there was no way to have both the address and the name. But I must be mistaking. It is valuable to have both the address and the name sometime, but not in line.

> (e.g in Settings)

in Setting it's not in line. (it shouldn't be)